src/offset/local/unix.rs

```diff
@@ -24,19 +24,25 @@ pub(super) fn offset_from_local_datetime(local: &NaiveDateTime) -> MappedLocalTi
 
 fn offset(d: &NaiveDateTime, local: bool) -> MappedLocalTime<FixedOffset> {
     TZ_INFO.with(|maybe_cache| {
-        maybe_cache.borrow_mut().get_or_insert_with(Cache::default).offset(*d, local)
+        maybe_cache.borrow_mut().offset(*d, local)
     })
 }
 
-// we have to store the `Cache` in an option as it can't
-// be initialized in a static context.
 thread_local! {
-    static TZ_INFO: RefCell<Option<Cache>> = Default::default();
+    static TZ_INFO: RefCell<Cache> = const { RefCell::new(
+        Cache {
+            zone: None,
+            source: Source::Uninitialized,
+            last_checked: SystemTime::UNIX_EPOCH,
+        }
+    ) };
 }
 
+#[derive(PartialEq)]
 enum Source {
     Environment { hash: u64 },
     LocalTime,
+    Uninitialized,
```

Comment:
Can we just represent this as `Option<Source>` instead? Seems more idiomatic to me.

Reply:
I'll update it to use an `Option`.